packages/core/echo/echo-db/src/echo-handler/clone.ts

```diff
@@ -85,7 +85,7 @@ const cloneInner = <T>(obj: EchoReactiveObject<T>, id: string): EchoReactiveObje
 const getObjectDoc = (core: ObjectCore): any => {
   let value = core.doc ?? core.docHandle!.docSync();
   for (const key of core.mountPath) {
-    value = value?.[key];
+    value = (value as any)?.[key];
```

Comment:
I sometimes do this temporarily to work through build errors, just making sure this is intentional because it seems dangerous inside core

Reply:
SpaceDoc

Reply:
Could be ObjectStructure also